src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/IncentiveReviewsServiceTest.kt

```diff
@@ -142,17 +142,18 @@ class IncentiveReviewsServiceTest {
         prisonId = "MDI",
       ),
     )
-    whenever(prisonApiService.getLocation(any())).thenReturnLocation("MDI-2-1")
+    whenever(prisonApiService.getLocation(any())).thenReturnLocation("MDI-2")
     whenever(offenderSearchService.getOffendersAtLocation(any(), any())).thenReturn(offenders)
     val nextReviewDatesMap = mapOf(
       110001L to LocalDate.now(clock).plusYears(1),
       110002L to LocalDate.now(clock).plusYears(1),
     )
     whenever(nextReviewDateGetterService.getMany(offenders)).thenReturn(nextReviewDatesMap)
 
-    val reviews = incentiveReviewsService.reviews("MDI", "MDI-2-1", "STD")
+    val reviews = incentiveReviewsService.reviews("MDI", "MDI-2-", "STD")
 
-    verify(offenderSearchService, times(1)).getOffendersAtLocation(any(), eq("MDI-2-1"))
+    verify(offenderSearchService, times(1)).getOffendersAtLocation(any(), eq("MDI-2-"))
+    verify(prisonApiService, times(1)).getLocation(eq("MDI-2"))
```

Comment:
…we want to look up the location trimming the trailing hyphen (else it's not a real location ID)